src/components/MonthCalendar.tsx

```diff
@@ -84,44 +87,30 @@ const MonthCalendar = ({ campaigns, events, onFocusDate, focusDate }: MonthCalen
                             <CalendarBar key={ c.id } campaign={ c } events={ campaignEvents } firstCalendarDay={ firstCalendarDay } firstMonthDay={ firstMonthDay } gridItems={ gridItems } month={ month } totalDaysInMonth={ totalDaysInMonth }/>
                         );
                     }) }
-                </div>
-                <div data-testid="calendar-wrapper" style={{
-                    display: 'grid',
-                    flexGrow: 1,
-                    gap: '0.5rem',
-                    gridTemplateColumns: 'repeat(7, minmax(125px, 1fr))',
-                    gridTemplateRows: `repeat(${calendarRows}, minmax(125px, 1fr))`,
-                }}>
+                </Box>
+                <Box data-testid="calendar-wrapper" display="grid" gridTemplateColumns="repeat(7, minmax(125px, 1fr))" gridTemplateRows={ `repeat(${calendarRows}, minmax(125px, 1fr))` } width={ 1 }>
                     { Array.from(Array(gridItems).keys()).map((_, index) => {
                         const currentDate = new Date(new Date(firstCalendarDay).setDate(firstCalendarDay.getDate() + index));
 
                         return (
-                            <div key={ index } data-testid={ `griditem-${index}` } style={{
-                                background: isInRange(currentDate, firstMonthDay, lastMonthDay) ? 'grey' : 'whitesmoke',
-                                position: 'relative',
-                            }}>
-                                <div style={{
-                                    top: 0,
-                                }}>
-                                    <FormattedDate
-                                        day="2-digit"
-                                        value={ currentDate }
-                                    />
-                                </div>
-                                <ul data-testid={ `day-${index}-events` } style={{
-                                    listStyle: 'none',
-                                    margin: 0,
-                                    padding:0 ,
-                                    width: '100%',
-                                }}>
+                            <Box key={ index } bgcolor={ isInRange(currentDate, firstMonthDay, lastMonthDay) ? grey[300] : grey[200] } data-testid={ `griditem-${index}` } m={ 0.5 } position="relative">
+                                <Typography>
+                                    <Box p={ 1 }>
```

Comment:
The docs said that the children prop is of 'node' type and it seems to apply to the content inside box even if not directly text. Don't know if this is standard use ==> I will place it inside Box here because it only applies to the text in this case, anyway, not multiple nodes.